plugins/modules/cloud/linode/linode_v4.py

```diff
@@ -87,8 +87,22 @@
     description:
       - The Linode API v4 access token. It may also be specified by exposing
         the C(LINODE_ACCESS_TOKEN) environment variable. See
-        U(https://developers.linode.com/api/v4#section/Access-and-Authentication).
+        U(https://www.linode.com/docs/api#access-and-authentication).
     required: true
+  stackscript_id:
+    description:
+      - The numeric ID of the StackScript to use when creating the instance.
+        See U(https://www.linode.com/docs/api/stackscripts/).
+    type: int
+    required: false
```

Comment:
```suggestion
    version_added: 1.3.0
```
`required: false` is not necessary.